typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,18 @@
+const fullName: string = 'Katelyn Ngo';
+const isCool: boolean = true;
+const totalPets: number = 1;
+
+console.log(' Value of Full Name:', fullName);
+console.log('Value of isCool:', isCool);
+console.log('Value of totalPets:', totalPets);
+console.log('typeof fullName:', typeof fullName);
+console.log('type of isCool:', typeof true);
```

Comment:
Should be using the variable for the typeof isCool and not the boolean value here. If you changed isCool to something else, like a string, then this will still log boolean to the console.